shell/pages/c/_cluster/settings/performance.vue

```diff
@@ -154,6 +187,20 @@ export default {
             />
           </div>
         </div>
+        <!-- Web Worker handles Schemas and Counts websocket subscriptions  -->
+        <div class="mt-40">
+          <h2>{{ t('performance.advancedWorker.label') }}</h2>
+          <p>{{ t('performance.advancedWorker.description') }}</p>
+          <Banner color="error" label-key="performance.advancedWorker.banner" />
+          <!-- ToDo: should look into saving this into a cookie at some point -->
```

Comment:
This is an older comment stemming from an iteration wherein I was always loading the basic-worker as default and then having to convert to advanced afterward because management settings hadn't been loaded yet. I'm not delaying loading the worker until management loads making this comment redundant. Something that'd have been caught in a final audit before marking the PR as ready for review.